doc/Protocols/pr-mess_x-road_message_protocol_for_rest.md

```diff
@@ -0,0 +1,1390 @@
+![](img/eu_regional_development_fund_horizontal_div_15.png "European Union | European Regional Development Fund | Investing in your future")
+
+---
+
+# X-Road: Message Protocol for REST
+**Technical Specification**
+
+Version: 0.9.0  
+22.03.2019
+
+---
+
+## Version history
+
+|Date       | Version | Description                                                                                                         | Author         |
+|:----------|:--------|:--------------------------------------------------------------------------------------------------------------------|:---------------|
+|22.03.2019 | 0.9.0	  | Initial Markdown documentation (see [Appendix 2](#appendix-2-document-change-history) for earlier document history) | Caro Hautamäki |
```

Comment:
I think it would be better to have the version history in one place. Probably here in the beginning of the document is the convention that we have in the other documents too.

Reply:
Moved version history from appendix to the begin